File: C/towerOfHanoi.c
#include<stdio.h>
#include<stdlib.h>

void towerOfHanoi(int n,char source,char destiation,char auxiliary){
    if(n == 1){
        printf("Move disk 1 from source %c to destiation %c \n",source,destiation);
        return;
    }
    towerOfHanoi(n-1,source,auxiliary,destiation);
    printf("Move dick %d from source %c to destiation %c \n",n,source,destiation);
    towerOfHanoi(n-1,auxiliary,destiation,source);
}

int main(void){
    int n = 3;
    towerOfHanoi(n,'A','B','C');
}

File: Python/towerOfHanoi.py
def towerOfHanoi(n,source,destination,auxiliart):
    if(n == 1):
        print("Move disk 1 from source {} to destination {}".format(source,destination))
        return
    towerOfHanoi(n-1,source,auxiliart,destination)
    print("Move disk {} from source {} to destination {}".format(n,source,destination))
    towerOfHanoi(n-1,auxiliart,destination,source)

if __name__ == "__main__":
    towerOfHanoi(3,'A','B','C')